privacyidea/static/components/token/views/token.import.html

```diff
@@ -69,12 +69,27 @@ <h3 translate>Import tokens from a file</h3>
         </p>
         <input class="form-control"
                placeholder="{{ 'Pre Shared Key' | translate }}"
-               ng-keypress="form.password =''"
+               ng-keypress="form.password = ''"
                ng-model="form.psk" minlength="0" maxlength="32">
         <input class="form-control"
                placeholder="{{ 'Password' | translate }}"
                ng-keypress="form.psk = ''"
                ng-model="form.password">
+        <div></div>
+        <div class="mt-5px">
+            <p class="help help-block" translate>
```

Comment:
@cornelinux It is unclear how translations are handled, should the developer add e.g. german translations in the push request?

Reply:
No :-)
Just mark it, that it should be translated.